packages/test-runner-visual-regression/src/visualRegressionPlugin.ts

```diff
@@ -109,6 +110,24 @@ export function visualRegressionPlugin(
             return visualDiffCommand(mergedOptions, screenshot, session.browser.name, payload.name);
           }
 
+          if (session.browser.type === 'selenium') {
```

Comment:
Since this branch is identical to the `webdriver` branch except for type it might be good to combine the branches and use a union type when defining `browser`...